### PR TITLE
Django 5.1 final release tasks

### PR DESCRIPTION
**Pull request by nessita**
See 3 commits for details:
* Updated docs-related model fixtures for 5.1 release.
* Updated robots.docs.txt for Django 5.1.
*  Updated download page for 5.1 release.

This follows previous work done in #1444, #1445 and #1446 for Django 5.0 release.

### CODE REVIEW

File: djangoproject/static/robots.docs.txt [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+Disallow: /pl/5.1/internals
+Disallow: /pl/5.1/ref
+Disallow: /pl/5.1/releases
+Disallow: /pl/5.1/topics
+Disallow: /pt-br/5.1/howto
+Disallow: /pt-br/5.1/internals
+Disallow: /pt-br/5.1/intro
+Disallow: /pt-br/5.1/ref
+Disallow: /pt-br/5.1/releases
+Disallow: /pt-br/5.1/topics
```
Comment (nessita) [inline djangoproject/static/robots.docs.txt:41]: I triple checked this output in two different checkouts/computers and also Sarah ran the command herself and got the same output.

**Review by bmispelon — APPROVED** [review]
These changes look entirely reasonable and follow what was done for the previous release. Thanks 👍🏻 

(I'll deploy this momentarily and post back here once that's done)

**Comment by bmispelon** [comment]
Deployed 🚀

**Comment by nessita** [comment]
> Deployed 🚀

Thank you so much! :star2:

**Pull request merged**